INSTALL.md

```diff
@@ -61,7 +74,7 @@ Other Fedora versions or distributions may require some adaptation.
 6. Test your local installation:
 
    ```bash
-   sudo PYTHONPATH=./src python3 -m subscription_manager.scripts.subscription_manager
+   sudo PYTHONPATH=src python3 -m subscription_manager.scripts.subscription_manager
```

Comment:
Why this change?

Reply:
The `./` does not bring any value, it’s implicit. Environment variables aren’t shell commands, where `./` is required to run an executable in the current directory.

Reply:
Reverted.